bin/nova_status.py

```diff
@@ -1,337 +0,0 @@
-#!/usr/bin/env python
```

Comment:
why we removed those scripts?

Reply:
I could not find who/what is using them and they are not defined as entry points